include/spm.rst

```diff
@@ -3,16 +3,25 @@
 Secure Partition Manager (SPM)
 ##############################
 
-The Secure Partition Manager (SPM) uses the SPU peripheral to configure security attributions for the flash, SRAM, and peripherals.
-Note that the SPU peripheral is the nRF version of an IDAU (Implementation-Defined Security Attribution Unit).
+The Secure Partition Manager (SPM) provides functionality for the Secure Zone of the nRF9160.
```

Comment:
It's fine that you made this a bit more generic, I would choose Secure execution environment, instead. Secure Zone is not an accepted terminology, AFAIK

Reply:
"Trusted Execution Environment" perhaps?

Reply:
I went with Trusted Execution Environment since it's more widespread AFAICT. Tell me if I've misunderstood the terminology.

Reply:
Yeap, that's fine.